pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -355,6 +356,15 @@ func (v Vertex) GetReplicas() int {
 	return int(*v.Spec.Replicas)
 }
 
+func (v Vertex) MapUdfStreamEnabled() (bool, error) {
+	if v.Spec.Metadata != nil && v.Spec.Metadata.Annotations != nil {
+		if mapUdfStream := v.Spec.Metadata.Annotations[MapUdfStreamKey]; mapUdfStream != "" {
```

Comment:
```golang
if mapUdfStream, existing := v.Spec.Metadata.Annotations[MapUdfStreamKey]; existing && mapUdfStream == "true" {
  return true
}
```

Reply:
Hmm, `strconv.ParseBool` actually covers a boarder [true cases](https://pkg.go.dev/strconv) which should be better?